core/src/create.jl

```diff
@@ -225,9 +236,10 @@ function Connectivity(db::DB, config::Config, chunk_size::Int)::Connectivity
     end
     flow .= 0.0
 
-    if config.solver.autodiff
-        # FixedSizeDiffCache performs better for sparse matrix
-        flow = FixedSizeDiffCache(flow, chunk_size)
+    chunk_sizes = get_chunk_sizes(config, chunk_size)
+    if !isempty(chunk_sizes)
```

Comment:
If my other comment is true, you could still do this, here and below: `if config.solver.autodiff`. That is easier to read than `isempty`.